delta/plugins/elasticsearch/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/ElasticSearchViewDecodingSpec.scala

```diff
@@ -49,13 +50,16 @@ class ElasticSearchViewDecodingSpec
     new ResolverContextResolution(rcr, (_, _, _) => IO.raiseError(ResourceResolutionReport()))
 
   implicit private val caller: Caller = Caller.Anonymous
-  private val decoder                 = ElasticSearchViewJsonLdSourceDecoder(uuidF, resolverContext)
+  private val decoder                 =
+    ElasticSearchViewJsonLdSourceDecoder(uuidF, resolverContext).runSyncUnsafe()
 
   "An IndexingElasticSearchViewValue" should {
     val mapping  = json"""{ "dynamic": false }""".asObject.value
     val settings = json"""{ "analysis": { } }""".asObject.value
 
     val indexingView = IndexingElasticSearchViewValue(
+      name = None,
```

Comment:
We could have an `apply` method that would allow to not having to pass a name and description in the tests
(as they only have value on the user perspective)